### DISCUSSION TITLE
Custom fields on order status emails

### DISCUSSION THREAD

**Discussion by timeverts** [craftcms/commerce#2341] [Ideas]
### Description
This is a feature request.

It would be great if you could add support for a field layout in Order Status Emails. This would make it possible to define CMS manageable components of an email that can be utilised inside the email template, other than just the order itself. For example, I could add an asset field to the order status email that would allow specifying the header image to use in the template for that order status email.

### Additional info

- Craft version: 3.3.3
- PHP version: 7.1
- Database driver & version: mySQL
- Plugins & versions: Commerce 2.1.13

**Comment by bossanova808** [comment]
We currently use a global to do this (which is fine for us as we use the same marketing banner on all emails at a particular time)...but I do generally have an aversion to globals & can see use cases where having a per-email asset might be useful.

(Just commenting to follow along!).
